src/Action/LtiInstance/LtiInstanceUpdateAction.php

```diff
@@ -79,7 +79,7 @@ public function __invoke(Request $updateActionRequest, string $ltiInstanceId): R
         /** @var LtiInstance $ltiInstance */
         $ltiInstance = $this->repository->find($ltiInstanceId);
 
-        if (!$ltiInstance) {
+        if ($ltiInstance == null) {
```

Comment:
Please use strict comparison operator to check if value is null
https://phpbestpractices.org/#checking-for-null

Reply:
fixed